docs/users_guide/extending_ghc.rst

```diff
@@ -425,7 +425,7 @@ will print out the name of any top-level non-recursive binding with the
     import Control.Monad (unless)
     import Data.Data
 
-    data SomeAnn = SomeAnn deriving (Data, Typeable)
```

Comment:
In this example code I'm removing the `deriving Typeable` bit because that no longer has any effect.